tests/test_models/test_engine/test_db_storage.py

```diff
@@ -14,16 +14,17 @@
 from models.review import Review
 from models.state import State
 from models.user import User
-import json
+import MySQLdb
 import os
 import pep8
-import unittest
+from unittest import TestCase
+from unittest import skipIf
 DBStorage = db_storage.DBStorage
```

Comment:
Merge with line 9 into `from models.engine.db_storage import DBStorage`.

Reply:
Just fixed with previous and to also import __ doc__